applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1135,30 +1200,20 @@ static void stream_disabled_cb(struct bt_bap_stream *stream)
 
 static void stream_stopped_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
-	int ret;
-
 	/* NOTE: The string below is used by the Nordic CI system */
 	LOG_INF("Stream %p stopped. Reason %d", (void *)stream, reason);
 
-	if (IS_ENABLED(CONFIG_BT_AUDIO_TX)) {
-		uint8_t device_index;
-
-		ret = device_index_get(stream->conn, &device_index);
-		if (ret) {
-			LOG_ERR("Device index not found");
-		} else {
-			ret = bt_le_audio_tx_stream_stopped(device_index);
```

Comment:
Could this just be removed now?

Reply:
No I think that was removed by mistake, will add it back